SRC/NBASIC/nbas111e.txt

```diff
@@ -202,55 +202,54 @@ the functions' description.
 
 3.2. USING VRAM AS LOGICAL SEGMENTS
 
-NestorBASIC allows you to use the VRAM for emulate extra RAM segments. If 
-NestorBASIC founds a number S of segments, segment numbers 0 to S-1 refers to 
-RAM segments, as explained above. But segments S to S+7 or S+3 (according to 
-the VRAM capacity, 128K or 64K) are also available, and they refer to VRAM.
+NestorBASIC allows you to use VRAM to emulate extra RAM segments. If NestorBASIC
+found a number S of segments, segments 0 to S-1 correspond to RAM segments,

```

Comment:
I think that this `found` should be `finds` instead.

Reply:
> 
Changed to: "If the number 
of segments found by NestorBASIC is S, then segments 0 to S-1 will correspond..."